projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -334,6 +334,50 @@ export class IgxDatePickerComponent extends PickerBaseDirective implements Contr
         this.valueChange.emit(this.dateValue);
         this._onChangeCallback(this.dateValue);
     }
+    /**
+     * The format used to display the picker's value when it's not being edited.
+     *
+     * @remarks
+     * Uses Angular's DatePipe.
+     *
+     * @example
+     * ```html
+     * <igx-date-picker displayFormat="EE/M/yy"></igx-date-picker>
+     * ```
+     *
+     */
+    @Input()
+    public override set displayFormat(value: string) {
+        if (this.dateTimeEditor) {
+            this.dateTimeEditor.displayFormat = value;
```

Comment:
Why did this switch away from direct bindings? Was this so the tests can check the inferred values? Because tests can also just query for the editor directive for checks